frontends/p4/alias.h

```diff
@@ -240,17 +237,19 @@ class ReadsWrites : public Inspector {
         rw.emplace(expression, result);
     }
 
-    const SetOfLocations *get(const IR::Expression *expression) {
-        expression->apply(*this);
+    const SetOfLocations *get(const IR::Expression *expression,
+                              const Visitor::Context *ctxt = nullptr) {
```

Comment:
nit: `ctxt` should probably not be optional here -- all callers of `get` would need to supply it.  Or if it is optional, the default should be `getChildContext()` rather than `nullptr`.

Reply:
Yes, certainly. It was `nullptr` during the migration of all users. Fixed now!